content/custom-commands/commands.md

```diff
@@ -0,0 +1,367 @@
+
+##### Contains
+
+Messages **containing** the trigger text (**4**) will trigger the command.
+
+##### Regex
+
+Messages matching the trigger text (**4**) as a **[regex pattern](/reference/regex)** will trigger the command.
+
+##### Exact Match
+
+Messages which **exactly** match the trigger text (**4**) will trigger the command.
+
+##### Reaction
+
+Reactions to a message will trigger the command.
+
+Can specify **Added Only**, **Removed Only**, or **Both** to restrict which types of Reactions will trigger the command.
+
+{{% notice style="tip" title="Filtering Emojis" %}}
+
+You cannot specify which emojis the command will trigger on. If you'd like to limit which emojis run the code, you will
+need to write that code yourself in the response.
+
+Example:
+
+```go
+  {{ if eq .Reaction.Emoji.APIName "😀" "⭐️" }}
+    This is an allowed reaction!
+  {{ else if eq .Reaction.Emoji.APIName "🦆" }}
+    This is not an allowed reaction.
+  {{ end }}
+```
+
+{{% /notice %}}
+
+##### Hourly/Minute Interval
+
+Interval triggers will run the command at a set interval in the selected channel.
+
+The Member in context is nil, and functions relying on the Member context will fail. However, `exec` will run commands
+as the bot.
+
+![Overview of interval configuration options.](interval_trigger_options.png?width=60vw)
+
+<center>
+
+**1** Interval **2** Channel **3** Excluding hours/weekdays
+
+</center>
+
+Interval (**1**) sets how often the command will run in **hours** or **minutes**.
+
+Channel (**2**) specifies a channel to run the command in. The response will be sent to this channel.
+
+Excluding hours and/or weekdays (**3**) prevents the command from triggering during those hours or weekdays. **This uses
+UTC time**, not your local timezone.
+
+When editing an interval command, a **Run Now** button appears at the bottom of the page. It executes the command as
+long as it isn't disabled and a channel is selected.
+
+Manually running an interval command reschedules all subsequent runs based off the current time.
+
+{{% notice info %}}
+
+You must specify a channel to run interval commands in even if the command doesn't output a message.
+
+{{% /notice %}}
+
+#### Case Sensitivity
+
+Any commands which allow you to specify trigger text (ex. Command, Regex, Exact match, etc.) have a **Case sensitivity**
+toggle (**5**) which is off by default. A case sensitive trigger `yagPDB` will trigger on "yagPDB" but not "yagpdb" or
+"YAGPDB".
+
+#### Edit Message Trigger
+
+Commands which trigger on messages have a **Trigger on message edits** toggle (**6**) which is off by default. If a
+message is edited and matches the trigger text, it will trigger the command.
+
+The edited message toggle is an _additional_ trigger to the sent message trigger. If you'd like to _only_ trigger on
+message edits, you will need to conditional branch with `{{ .IsMessageEdit }}`.
+
+This feature is [premium only](/premium).
```

Comment:
Not entirely sure if it might be worthwhile to move this one up a bit,
perhaps right underneath this very header?